.gitignore

```diff
@@ -3,3 +3,4 @@
 
 .DS_Store
 .byebug_history
+.env
```

Comment:
Let's add the  dump.db here in the gitignore as Jorge  says

Reply:
Done